bot/cogs/snakes.py

```diff
@@ -42,8 +45,12 @@ def __init__(self, bot: AutoShardedBot):
         """
 
     # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+    @command()
+    async def play(self, ctx: Context, order):
+        if order == "left":
+            await ctx.send(self.game)
 
 
 def setup(bot):
-    bot.add_cog(Snakes(bot))
+    bot.add_cog(Snakes(bot, SnakeGame((5, 5))))
```

Comment:
This is quite weird - maybe you should instantiate it in the `__init__` instead?